Description: Fixes broken conf.py
 The upstream conf.py makes the sphinx-build call fail and stack dump.
 This patch removes the most recent changes that appeared a few days ago,
 which broke sphinx documentation generation.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-09-17

--- python-oslo.vmware-0.6.0.orig/doc/source/conf.py
+++ python-oslo.vmware-0.6.0/doc/source/conf.py
@@ -39,14 +39,14 @@ exclude_patterns = [
     'api/oslo.vmware._*',  # skip private modules
 ]
 
-# Prune the excluded patterns from the autoindex
-for line in fileinput.input('api/autoindex.rst', inplace=True):
-    found = False
-    for pattern in exclude_patterns:
-        if fnmatch.fnmatch(line, '*' + pattern[4:]):
-            found = True
-    if not found:
-        print line,
+## Prune the excluded patterns from the autoindex
+#for line in fileinput.input('api/autoindex.rst', inplace=True):
+#    found = False
+#    for pattern in exclude_patterns:
+#        if fnmatch.fnmatch(line, '*' + pattern[4:]):
+#            found = True
+#    if not found:
+#        print line,
 
 # The suffix of source filenames.
 source_suffix = '.rst'
